source/usage-examples/command.txt

```diff
@@ -4,16 +4,52 @@ Run a Command
 
 .. default-domain:: mongodb
 
-You can run all raw database operations using the
-``MongoDatabase.runCommand()`` method. A raw database operation is a
-command you can execute directly on the MongoDB server CLI. These
-commands include administrative and diagnostic tasks, such as fetching
-server stats or initializing a replica set. Call the ``runCommand()``
-method with a Bson command object on an instance of a ``MongoDatabase``
-to run your raw database operation.
+You can run commands directly on the MongoDB server CLI using the
+``RunCommand()`` method. 
+
+The following example passes a document containing the ``"dbStats"``
+command to the ``RunCommand()`` method, which requests for statistics
+about the ``sample_restaurants`` database: 
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/command.go
+   :start-after: begin runCommand
+   :end-before: end runCommand
+   :emphasize-lines: 5
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/command.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, the ``result`` variable should
+contain the following: 
```

Comment:
```suggestion
contain the following cursor (maybe struct?): 
```
Suggestion: Could this be more specific? I am not sure what I am looking at, and needed to consult API docs to try and figure it out.